packages/flutterfire_cli/lib/src/commands/config.dart

```diff
@@ -32,6 +35,22 @@ import '../flutter_app.dart';
 
 import 'base.dart';
 
+final _androidBuildGradleRegex = RegExp(
```

Comment:
Is it worth leaving some regex101 links here to demonstrate the purpose of the regex to contributors?

Reply:
I usually do but I now use `Regex Workbench` extension in vscode now for this 🙈 can add some though in post cleanup (gonna add comments as well)